apps/cookbook/src/app/showcase/calendar-showcase/calendar-showcase.component.ts

```diff
@@ -103,4 +102,19 @@ export class CalendarShowcaseComponent {
       inputValues: ['"local" | "UTC"'],
     },
   ];
+
+  events: ShowcaseProperty[] = [
+    {
+      name: 'dateSelect',
+      description: 'Emitted when a date is clicked.',
+      inputValues: ['EventEmitter<void>'],
```

Comment:
```suggestion
      inputValues: ['EventEmitter<Date>'],
```

Reply:
Should be resolved with the new commits 😄